services/QuillCMS/spec/workers/bulk_reindex_responses_worker_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe BulkReindexResponsesWorker do
+  let(:worker) { described_class.new }
+  let!(:response) { create(:response, question_uid: 'blah', count: 1, first_attempt_count: 1) }
+  let!(:response2) { create(:response, question_uid: 'blah', parent_id: response.id, count: 1, first_attempt_count: 1) }
```

Comment:
More of something to keep in mind in the future. 

If I'm using creating the same factory object multiple times, I'll usually use `create_list` to cut down on boilerplate and you have an ordered object if you need it:

`let!(:responses) { create_list(:response, 4, question_uid: 'blah', parent_id: response.id, count: 1, first_attempt_count: 1) }`